systemtest/010-inspect.bats

```diff
@@ -27,11 +27,16 @@ load helpers
     # Now run inspect locally
     run_skopeo inspect dir:$workdir
     inspect_local=$output
+    run_skopeo inspect --raw dir:$workdir
+    inspect_local_raw=$output
+    config_digest=$(jq -r '.config.digest' <<<"$inspect_local_raw")
 
     # Each SHA-named file must be listed in the output of 'inspect'
     for sha in $(find $workdir -type f | xargs -l1 basename | egrep '^[0-9a-f]{64}$'); do
-        expect_output --from="$inspect_local" --substring "sha256:$sha" \
-                      "Locally-extracted SHA file is present in 'inspect'"
+        if [ "sha256:$sha" != "$config_digest" ]; then
```

Comment:
As a favor for ignorami like me, could you pretty-please add a comment explaining the circumstances in which this will trigger?

Reply:
Added